test/cases/testdata/strings/test-strings-0925.yaml

```diff
@@ -0,0 +1,13 @@
+cases:
+- note: 'strings/repeat_test'
+  modules:
+  - |
+    package test
+
+    p = x {
+      x := strings.repeat("hi", -1)
+    }
+  query: data.test.p = x
+  note: 'count must be greater than zero'
+  want_result:
+  - x: "" 
```

Comment:
Something like [this](https://github.com/open-policy-agent/opa/blob/main/test/cases/testdata/replacen/test-replacen-bad-operands.yaml) test would be helpful.